src/lib/project.js

```diff
@@ -39,16 +41,19 @@ async function project(name) {
 
   await step('Initialize Git repo', 'git init -q');
 
-  // `/dev/null` is the only way to silence Husky's install log msg.
-  await step('NPM install', 'npm ci --silent > "/dev/null" 2>&1');
+  // `/dev/null` on linux or 'NUL' on windows is the only way to silence Husky's install log msg.
+  await step(
+    'NPM install',
+    `npm ci --silent > ${isWindows ? 'NUL' : '"/dev/null" 2>&1'}`
+  );
 
   // process.cwd() is full path to user's terminal + path/to/name.
   await setProjectName(process.cwd());
 
   // `-n` (no verify) skips Husky's pre-commit hooks.
   await step(
     'Git init commit',
-    `git add . && git commit -m 'Init commit' -q -n && git branch -m main`
+    `git add . && git commit -m "Init commit" -q -n && git branch -m main`
```

Comment:
nit: we could use single quotes on the outside (for consistency since var interpolation isn't used here)